README.md

```diff
@@ -19,9 +19,21 @@ OCaml / Reason types (have a look in [`src/shared`](./src/shared)).
 
 ## Running
 
+### Build the front-end
+
 - `yarn && yarn build && yarn bundle`
+
+### Build and run the backend 
+
+_it's a quiet startup; check http://localhost:8080/graphql to see that it's running!_
+
 - Make sure you have `opam` (version 2) installed and are on a switch >= 4.03 (`ocaml-graphql-server` requires at least OCaml 4.03)
 - `opam install . --deps-only`
+- At the moment we use some cutting-edge features of `ocaml-graphql-server` so, switch to another directory and [pin](https://opam.ocaml.org/doc/Usage.html#opam-pin) the package (revision bd121b2f8edd53ccaa4a6e8b8377c8e85cd40935 works)
+- `git clone git@github.com:andreas/ocaml-graphql-server`
+- `cd ocaml-graphql-server`
+- `opam pin add graphql-lwt .`
+- `opam pin add graphql .`
```

Comment:
FWIW, instead of these 4 lines, you can as well do:
```
opam pin add graphql-lwt --dev-repo
opam pin add graphql --dev-repo
```

this (a) does not require a local checkout of the graphql repository, and (b) uses the `dev-repo` information of the `graphql` and `graphql-lwt` packages of the opam-repository. :)

Reply:
That's really good to know. Thanks!